pkgs/by-name/en/enry/package.nix

```
@@ -0,0 +1,57 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  versionCheckHook,
+  writeScript,
+}:
+
+let
+  commitHash = "f10711437bfbb25b15506eb69dde24bb7decd222"; # matches tag release
+in
+buildGoModule rec {
+  pname = "enry";
+  version = "1.3.0";
+
+  src = fetchFromGitHub {
+    owner = "go-enry";
+    repo = "enry";
+    tag = "v${version}";
+    hash = "sha256-JSYOCdWZnCM4Kq3EFkS7XZ5n0ahkR+A8omI4z1S45z8=";
+  };
+
+  vendorHash = "sha256-tUDhpeXlKpo1jnNyk0U3CcXroOlv7lHVUxc1wAnysG4=";
+
+  ldflags = [
+    "-X main.version=${version}"
+    "-X main.commit=${commitHash}"
```

```suggestion
    "-X main.commit=${finalAttrs.src.rev}"
```
You can access the commit hash like that

---

Oh, didn't know that about revs. Okay then, keep it like that